battleship-service/src/main/java/dreamteam/battleship/service/navigator/HtmlNavigator.java

```diff
@@ -39,10 +39,11 @@ public void init() throws ServletException {
 
         } catch (ParseException e) {
             logger.debug(e);
+            throw new ServletException();
         } catch (Exception e) {
             logger.debug(e);
+            throw new ServletException();
```

Comment:
Why are we repackaging any Exception into Servlet one? Without message? And why the different catch blocks for same handling?